plugin/storage/es/mappings/gen_assets.go

```diff
@@ -213,7 +213,7 @@ var _escData = map[string]*_escFile{
 		name:    ".nocover",
```

Comment:
consider using the new `embed` package (see #2749)

Reply:
Used embed instead of esc. Would remove corresponding make target dependency as well.